src/composables/multisigTransactions.ts

```diff
@@ -42,8 +43,44 @@ export function useMultisigTransactions({ store }: UseMultisigTransactionsOption
     });
   }
 
-  async function getTransactionByHash(txHash: string) {
-    return fetchJson(`${activeNetwork.value.multisigBackendUrl}/tx/${txHash}`);
+  /**
+ * Get raw proposed transaction from multisig backend
+ * @param {string} txHash
+ * @returns {IRawMultisigTx | null} returns transaction if exists or null
+ */
+  async function getTransactionByHash(txHash: string): Promise<IRawMultisigTx | null> {
```

Comment:
I suggest to rename this function from `get...` to `fetch...` as this is not a getter (getters are not asynchronous).